ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -206,6 +206,9 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         }
     }
 
+    private var attributedTitleFont: UIFont?
```

Comment:
Nit: we try to keep our file organized as per the Microsoft Swift docs here: [File Organization
](https://microsoft.github.io/swift-guide/FileOrganization.html).

Specifically, `private` vars are down lower in the file, while `open` and `public` is higher up.

Reply:
Thanks, I would usually have these at the end of the file, but wanted to group them with existing private vars. I can move them to the end of the class though, thanks!